newrelic.js

```diff
@@ -44,6 +44,38 @@ exports.config = {
    * attributes include/exclude lists.
    */
   allow_all_headers: true,
+  application_logging: {
+    /**
+     * If this is disabled, all sub-features are disabled.
+     * If it is enabled, the individual sub-feature configurations take effect.
+     */
+    enabled: true,
+    forwarding: {
+      /**
+       * Toggles whether the agent gathers log records for sending to New Relic.
+       */
+      enabled: false,
+      /**
+       * Number of log records to send per minute to New Relic.
+       * Controls the overall memory consumption when using log forwarding.
+       */
+      max_samples_stored: 10000
+    },
+    metrics: {
+      /**
+       * Toggles whether the agent gathers the the user facing `Logging/lines`
+       * and `Logging/lines/{SEVERITY}` *Logging Metrics* used in the Logs chart
+       * on the APM Summary page.
+       */
+      enabled: true
```

Comment:
this is currently defaulted to true there's no reason to set this in here

Reply:
The ticket specifically identifies `application_logging.forwarding.enabled` to be set to true and described. Have I read this wrong?